### DISCUSSION TITLE
Paddle kie训练后测试，单跑det和rec都没问题，ser的图像长图只能识别一半不到的尺寸，ser识别尺寸在哪里调整

### DISCUSSION THREAD

**Discussion by Schawn** [PaddlePaddle/PaddleOCR#12896] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：windows10
-  PaddleOCR：2.7  问题相关组件/Related components：Paddle kie训练后测试，单跑det和rec都没问题，ser的图像长图只能识别一半不到的尺寸，ser识别尺寸在哪里调整
- 运行指令/Command Code：
- python tools/infer_kie_token_ser.py -c configs/kie/vi_layoutxlm/ser_vi_layoutxlm_xfund_zh.yml  -o Architecture.Backbone.checkpoints=./output/ser_vi_layoutxlm_xfund_zh0911fapiao/best_accuracy Global.infer_img="./train_data/1/"
Global.infer_mode=True Global.kie_rec_model_dir='./output/rec_db_inference1/Student' Global.kie_det_model_dir='./output/det_db_inference20230808'

- 完整报错/Complete Error Message：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/50742027/688464ed-446f-4d4a-a8b5-d96d1f2a9ce0)

**Comment by jackieZhouQQ** [comment]
@Schawn 遇到了一样的问题，你解决了吗？

**Comment by Schawn** [comment]
完全没办法解决，只能尝试按最后识别的一个元素切割图片二次识别

jackieZhouQQ ***@***.***>于2023年10月11日 周三18:44写道：

**Comment by jackieZhouQQ** [comment]
这样感觉好呆啊 😂



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月11日(周三) 晚上6:48
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PaddlePaddle/PaddleOCR] Paddle kie训练后测试，单跑det和rec都没问题，ser的图像长图只能识别一半不到的尺寸，ser识别尺寸在哪里调整 (Issue #10957)




 完全没办法解决，只能尝试按最后识别的一个元素切割图片二次识别 
 
 jackieZhouQQ ***@***.***&gt;于2023年10月11日 周三18:44写道： 
 
 &gt; @Schawn <https://github.com/Schawn&gt; 遇到了一样的问题，你解决了吗？ 
 &gt; 
 &gt; — 
 &gt; Reply to this email directly, view it on GitHub 
 &gt; <https://github.com/PaddlePaddle/PaddleOCR/issues/10957#issuecomment-1757388976&gt;, 
 &gt; or unsubscribe 
 &gt; <https://github.com/notifications/unsubscribe-auth/AMDEGC4374WEUC2YHIZRKQ3X6Z2BJANCNFSM6AAAAAA5BC4VSA&gt; 
 &gt; . 
 &gt; You are receiving this because you were mentioned.Message ID: 
 &gt; ***@***.***&gt; 
 &gt;

**Comment by jackieZhouQQ** [comment]
训练集也这样切割处理吗？整体效果还行嘛



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月11日(周三) 晚上6:48
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PaddlePaddle/PaddleOCR] Paddle kie训练后测试，单跑det和rec都没问题，ser的图像长图只能识别一半不到的尺寸，ser识别尺寸在哪里调整 (Issue #10957)




 完全没办法解决，只能尝试按最后识别的一个元素切割图片二次识别 
 
 jackieZhouQQ ***@***.***&gt;于2023年10月11日 周三18:44写道： 
 
 &gt; @Schawn <https://github.com/Schawn&gt; 遇到了一样的问题，你解决了吗？ 
 &gt; 
 &gt; — 
 &gt; Reply to this email directly, view it on GitHub 
 &gt; <https://github.com/PaddlePaddle/PaddleOCR/issues/10957#issuecomment-1757388976&gt;, 
 &gt; or unsubscribe 
 &gt; <https://github.com/notifications/unsubscribe-auth/AMDEGC4374WEUC2YHIZRKQ3X6Z2BJANCNFSM6AAAAAA5BC4VSA&gt; 
 &gt; . 
 &gt; You are receiving this because you were mentioned.Message ID: 
 &gt; ***@***.***&gt; 
 &gt;
